src/main/java/uk/gov/digital/ho/pttg/application/namematching/candidates/AliasSurnameCombinationsFunctions.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.digital.ho.pttg.application.namematching.candidates;
+
+import uk.gov.digital.ho.pttg.application.namematching.CandidateName;
+import uk.gov.digital.ho.pttg.application.namematching.InputNames;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.Collections.reverse;
+
+final class AliasSurnameCombinationsFunctions {
+
+    private AliasSurnameCombinationsFunctions() {
+        throw new UnsupportedOperationException("Helper class for AliasSurnameCombinationsFunctions containing only static methods - no need to instantiate.");
+    }
+
+    static List<String> removeName(String nameToRemove, List<String> names) {
+        return names.stream()
+                .filter(name -> !name.equals(nameToRemove))
+                .collect(Collectors.toList());
+    }
```

Comment:
Is this a good candidate for List.remove?

Reply:
Yes - I've made the change